File: src/SolucionesJava/Solucion_11.java
package SolucionesJava;

import java.util.Scanner;

public class Solucion_11 {
    public static void solucion_11() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("introduce un numero para ver su secuencia de Fibonacci:");
        int numero = scanner.nextInt();
        System.out.println("introduce el numero de repeticiones de la secuencia que desea:");
        int totalSecuencia = scanner.nextInt();

        int anterior = numero;
        int actual = numero;

        for (int i = 1; i <= totalSecuencia; i++) {
            System.out.println(anterior);
            int siguiente = anterior + actual;
            anterior = actual;
            actual = siguiente;
        }
    }
}

File: src/SolucionesKotlin/Solucion_11.kt
package SolucionesKotlin

fun Solucion_11 (){
    println("introduce un numero para ver su secuencia de Fibonacci: ")
    val numero = readln().toInt()
    println("introduce el numero de repeticiones de la secuencia que desea: ")
    val totalSecuencia = readln().toInt()
    var anterior = numero
    var actual = numero
    for (i in 1..totalSecuencia) {
        println(anterior)
        val siguiente = anterior + actual
        anterior = actual
        actual = siguiente
    }
}